packages/create-bison-app/template/components/LoginForm.tsx

```diff
@@ -51,9 +51,14 @@ export function LoginForm() {
     try {
       setIsLoading(true);
       const { data } = await login({ variables: formData });
+
+      if (!data?.login?.token) {
+        throw new Error('Login failed.');
```

Comment:
It's probably out of the scope of this PR, but we should create another issue to display some kind of user feedback when this is error is thrown.